chart/kepler/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-name:  kepler
+name: kepler
```

Comment:
@bradmccoydev relocating the `Chart.yaml` causes the integration test failure, can you fix that?

Reply:
sure I have added the default working directory for the action and it will now run in the new folder structure. 

I have also improved the action to run a test again if someone pushes back to the PR to ensure that their change still works:

```yaml
on:
  pull_request:
    types:
      - "opened"
      - "edited"
      - "synchronize"
```

I tested this on my test repo also.